docs/VnVPlan/VnVPlan.tex

```diff
@@ -392,9 +393,44 @@ \subsubsection{Module ?}
 
 \end{enumerate}
 
-\subsubsection{Module ?}
+\subsubsection{Usability and Humanity Requirements}
+\begin{enumerate}
 
-...
+	\item{NFRT-UH-1\\}
+
+	Type: Dynamic, Manual
+
+	Initial State: Device is connected to the internet and application is not open
+
+	Input/Condition: The user launches the application on Google Chrome
+
+	Output/Result: The system can be assessed through Google Chrome
+
+	How test will be performed: The tester will attempt to launch the application on the Google Chrome
+	web browser and using a device that is connected to the internet.
+
+	\item{NFRT-UH-2\\}
+
+	Type: Dynamic, Manual
+
+	Initial State: Device is connected to the internet and the application is open
+
+	Input/Condition: User disconnects from the internet
+
+	Output/Result: The system notifies the user that there is no network connection and functionality
+	will be limited.
+
+	How test will be performed: The tester disconnect from the internet while the application is open.
+
+	\item{NFRT-UH-3\\}
+
+	Type: Dynamic, Manual
+
+	How test will be performed: The testers will complete the manual system tests for functional
+	requirements listed section \ref{Tests for Functional Requirements} on a MacOS desktop/laptop
+	device, a Windows desktop/laptop, an iOS mobile device, and an android mobile device.
```

Comment:
Which requirement is this referencing? I don't think this if for UH3. I think we should follow the format of `NFRT-UH3` to make traceability clearer (and my work later of creating the traceability matrix easier).

Reply:
For some requirements, I'm guessing we might want more than one test; or one test might cover more than one requirement. I've added a comment for which requirement each non-functional test corresponds to.